src/app/index.js

```diff
@@ -15,11 +17,7 @@ const PLUGIN_PREFIX = 'danger-plugin-'
 
 function makeGeneratorName(name) {
   name = _.kebabCase(name)
-
-  if (name.indexOf(PLUGIN_PREFIX) === 0) {
-    this.log(
```

Comment:
Hmm yeah I think that makes sense - `this` doesn't refer to what we think it does, since this function is outside of the `Generator` subclass.